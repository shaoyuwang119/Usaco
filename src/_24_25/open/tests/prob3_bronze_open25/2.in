50 50
ooooowydsaohzqomcketurctvouwbziesvelfuesoasxtgyugt
5 47
37 39
3 50
1 18
10 12
5 48
1 46
1 48
8 40
15 50
1 46
13 38
2 49
1 50
2 49
1 49
7 9
9 47
8 49
1 17
1 49
4 50
2 43
2 18
3 49
3 48
31 33
4 49
1 16
1 43
3 6
4 43
4 50
4 33
4 49
1 50
1 50
5 19
13 49
4 18
2 16
8 47
2 50
2 17
5 49
7 46
10 49
1 44
1 20
4 18
