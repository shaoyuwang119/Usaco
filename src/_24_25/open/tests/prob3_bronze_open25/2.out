399
-1
546
20
-1
462
483
546
221
252
483
154
525
598
525
546
-1
315
399
20
546
520
340
20
504
504
-1
483
20
360
-1
300
520
195
483
598
598
20
294
20
20
336
572
20
462
357
357
360
20
20
